### PR TITLE
Description of virtio block device

### PR DESCRIPTION
**Pull request by gautam-meeshi**

### CODE REVIEW

File: Block_Device.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+VirtIO Devices - These are the abstraction layer that enable VM to use the hardware like disk, network on the host.
```
Comment (SatyamModi) [inline Block_Device.md:1]: Testing

Comment (Gamma-Zero) [inline Block_Device.md:1]: Testing

Comment (Prakhar314) [inline Block_Device.md:1]: Testing

**Comment by chintansheth1711** [comment]
Reviewing this PR.

File: Block_Device.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+The main task of VirtIO devices is to communicate the data between VM and host.
```
Comment (souraavv) [inline Block_Device.md:2]: Testing

File: Block_Device.md (hunk 1 above)
Comment (chintansheth1711) [inline Block_Device.md:1]: Testing

File: Block_Device.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+Three main data structures are maintained to facilitate this communication.
```
Comment (SatyamModi) [inline Block_Device.md:4]: Kindly add more info about these data structures. Looks incomplete.

**Review by souraavv — APPROVED** [review]

**Review by ronakladhar — APPROVED** [review]

**Review by ronakladhar — APPROVED** [review]

**Pull request closed without merging**